src/actions/actionTypes.js

```diff
@@ -1,7 +1,13 @@
 const actionTypes = {
-    SIGNUP: 'SIGNUP',
-    LOGIN: 'LOGIN',
-    SIGNUPFAIL: 'SIGNUPFAIL',
-    SIGNUPSUCCESS: 'SIGNUPSUCCESS'
-}
+  SIGNUP: "SIGNUP",
+  LOGIN: "LOGIN",
+  SIGNUPFAIL: "SIGNUPFAIL",
+  SIGNUPSUCCESS: "SIGNUPSUCCESS"
+};
 export default actionTypes;
+
+export const CREATE_ARTICLE = "CREATE_ARTICLE";
```

Comment:
Define all the actions as key-value pairs inside the `actionTypes` object.

Reply:
@kafuuma Could you address this?